template/visual-diff/integration/visualdiff.spec.js

```diff
@@ -1,8 +1,13 @@
 describe('check the site for visual regressions', () => {
   beforeEach(() => {
     cy.eyesOpen({
-      appName: process.env.URL || 'localhost test',
-      batchName: Cypress.env('APPLITOOLS_BATCH_ID'),
+      appName: process.env.SITE_NAME || 'localhost-test',
+      batchName: process.env.SITE_NAME || 'localhost-test',
+      testName: 'Visual Diff',
```

Comment:
My bad. I was not aware that `testName` should be unique, so I would put the path of the page here, just like the `tag` in `eyesCheckWindow`.

Reply:
I'm not sure how to get this in the `beforeEach` call. the routes are only available in each test — can I set the test name from the test?

Reply:
I was able to leave this off — it uses the Cypress test name if `testName` isn’t set, so I made it `Visual Diff for ${route}`